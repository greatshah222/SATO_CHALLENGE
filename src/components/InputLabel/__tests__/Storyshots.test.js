import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({ storyKindRegex: /^InputLabel$/ });
