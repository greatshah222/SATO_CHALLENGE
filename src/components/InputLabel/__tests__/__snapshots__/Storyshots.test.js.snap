// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots InputLabel error 1`] = `
<div>
  <label
    className="MuiFormLabel-root MuiInputLabel-root InputLabel-root-64 MuiInputLabel-animated Mui-error Mui-error"
  >
    Standard label
  </label>
</div>
`;

exports[`Storyshots InputLabel standard 1`] = `
<div>
  <label
    className="MuiFormLabel-root MuiInputLabel-root InputLabel-root-22 MuiInputLabel-animated"
  >
    Standard label
  </label>
</div>
`;
